public/resources/core/js/controllers/inventory-product.controller.js

```diff
@@ -47,12 +47,16 @@
         vm.size = -1;
 
         vm.showFilters = showFilters;
-        vm.getElementsLength = getElementsLength;
+        vm.getElementsLength = getElementsLength;	
 
-        $scope.$watch(function () {
-            return vm.offset;
-        }, function () {
-            vm.list.status = LOADING;
+	$scope.$on(Party.EVENTS.USER_SESSION_SWITCHED, function (event, message, obj) {
+	    if (Party.isOrganization() || User.loggedUser.currentUser.id === User.loggedUser.id){
```

Comment:
Is possible for this if to be false? I mean can you be logged without being an organization and you current user being different from your logged user?

Not a critic with the if, just curious